### PR TITLE
FUT1-17704 - new codeSystem and ValueSet

### PR DESCRIPTION
**Pull request by sys-alex-matic**
Added CodeSystem and ValueSet for program.
Updated description for ehealth-usage-context-type. Updated IG intro for some resources regaring useContext.

### CODE REVIEW

File: input/resources/ValueSet-ehealth-ehealth-usage-context-type.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
...
...                              | Supports          | The type of activity supported by a module/element having this use context (see http://hl7.org/fhir/metadatatypes.html#UsageContext). For this context type, the value should be a code taken from the http://ehealth.sundhed.dk/vs/activitydefinition-code value set. |\n| range               | http://ehealth.sundhed.dk/cs/ehealth-usage-context-type                                    | Range             | The type of reference range with which it is relevant to use a module/element having this use context (see http://hl7.org/fhir/metadatatypes.html#UsageContext). For this context type, the value should be a code taken from the http://ehealth.sundhed.dk/vs/reference-range-type value set. |\n| questionnaire-related | http://ehealth.sundhed.dk/cs/ehealth-usage-context-type                                    | Questionnaire Related | The type of questionnaire element with which it is relevant to use a coded values/id registries having this use context (see http://hl7.org/fhir/metadatatypes.html#UsageContext). For this context type, the value should be a code taken from the http://ehealth.sundhed.dk/vs/questionnaire-elements value set. |\n",
```
Comment (sys-alex-matic) [inline input/resources/ValueSet-ehealth-ehealth-usage-context-type.json:23]: Tabellen kan ses på https://build.fhir.org/ig/fut-infrastructure/implementation-guide/branches/FUT1-17704_CCR0232S01_Terminology_CodeSystem_and_ValueSet/ValueSet-ehealth-usage-context-type.html.
@esn-systematic ønsker vi kun at vise focus og program som er forskellig fra expansion tabellen?

File: input/resources/ValueSet-ehealth-program.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      "name": "ehealth.sundhed.dk",
+      "telecom": [
+        {
+          "system": "url",
+          "value": "http://ehealth.sundhed.dk/terminology"
+        }
+      ]
+    }
+  ],
+  "description": "A list of program/project.",
```
Comment (sysMATGS) [inline input/resources/ValueSet-ehealth-program.json:23]: plurals

File: input/resources/ValueSet-ehealth-program.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
+{
+  "resourceType": "ValueSet",
+  "id": "ehealth-program",
+  "url": "http://ehealth.sundhed.dk/vs/ehealth-program",
+  "version": "2024-11-28T09:00:00",
+  "name": "Program",
+  "title": "Program",
+  "status": "active",
+  "experimental": true,
```
Comment (sysMATGS) [inline input/resources/ValueSet-ehealth-program.json:9]: experimental = false

File: input/resources/CodeSystem-ehealth-program.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
+{
+  "resourceType": "CodeSystem",
+  "id": "ehealth-program",
+  "url": "http://ehealth.sundhed.dk/cs/ehealth-program",
```
Comment (sys-alex-matic) [inline input/resources/CodeSystem-ehealth-program.json:4]: @esn-systematic do we want "ehealth-program" or just "program". it seems like we some times append "ehealth" and sometimes not.

File: input/resources/ValueSet-ehealth-program.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
+{
+  "resourceType": "ValueSet",
+  "id": "ehealth-program",
+  "url": "http://ehealth.sundhed.dk/vs/ehealth-program",
```
Comment (sys-alex-matic) [inline input/resources/ValueSet-ehealth-program.json:4]: Burde det være plural? "programs".

File: input/resources/ValueSet-ehealth-ehealth-usage-context-type.json (hunk 1 above)
Comment (esn-systematic) [reply input/resources/ValueSet-ehealth-ehealth-usage-context-type.json:23]: Som diskuteret er det ok at vise focus, program og vores egne.

File: input/resources/ValueSet-ehealth-program.json (hunk 2 above)
Comment (esn-systematic) [reply input/resources/ValueSet-ehealth-program.json]: Programs or projects using the eHealth Infrastructure and wanting to mark various resources as intended for their non-exclusive use.

File: input/resources/ValueSet-ehealth-program.json (hunk 3 above)
Comment (esn-systematic) [reply input/resources/ValueSet-ehealth-program.json]: name eHealthProgram
title eHealth Program

File: input/resources/CodeSystem-ehealth-program.json (hunk 4 above)
Comment (esn-systematic) [reply input/resources/CodeSystem-ehealth-program.json:4]: This is about particular use of the infrastructure, hence ehealth-program (similar to co-existence tag ehealth-system).

File: input/resources/ValueSet-ehealth-program.json (hunk 5 above)
Comment (esn-systematic) [reply input/resources/ValueSet-ehealth-program.json:4]: Either will work for the ValueSet, suggest singular.

File: input/resources/CodeSystem-ehealth-program.json [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+    {
+      "telecom": [
+        {
+          "system": "url",
+          "value": "http://ehealth.sundhed.dk/terminology"
+        }
+      ]
+    }
+  ],
+  "description": "A program/project of work for which this artifact is applicable.",
```
Comment (esn-systematic) [inline input/resources/CodeSystem-ehealth-program.json:22]: description along the lines of VS is better:
Codes for programs or projects using the eHealth Infrastructure and wanting to mark various resources as intended for their non-exclusive use.

**Review by esn-systematic — APPROVED** [review]

**Pull request merged**